pkg/grpc/grpc.go

```diff
@@ -112,6 +112,15 @@ func (g *Manager) GetGRPCConnection(address, id string, namespace string, skipTL
 	defer cancel()
 
 	dialPrefix := GetDialAddressPrefix(g.mode)
+	if sslEnabled {
+		dialPrefix = "https://"
```

Comment:
No we do not need `https://` scheme prefix. https:// is not eligible scheme for grpc client.
Even if Kestrel server (HTTP/2) is used as gRPC transport layer, gRPC client handles gRPC protocol.

Reply:
Changed.